cypress/page-objects/settings-page.js

```diff
@@ -0,0 +1,13 @@
+export class SettingsPage {
+    navigateToSettingsPage(){
```

Comment:
Is this being used or do you always use `cy.clickSettingsButton()` directly?

Reply:
fixed - moved the method `navigateToSettingsPage()` to searchPage class and use that in the tests

Reply:
moved the method `navigateToSettingsPage()`to `searchPage.js` class and use that in the tests